libs/input/src/index.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  FormControl,
+  FormErrorMessage,
+  FormHelperText,
+  FormLabel,
+  Input as ChakraInput,
+  InputGroup,
+  InputRightElement,
+} from '@chakra-ui/react';
+import React, { ChangeEvent } from 'react';
+
+import { IconClose } from '@millicast-react/dolbyio-icons';
+
+import { InputProps } from './types';
+
+const Input = ({ disabled, error, label, helper, onChange, placeholder, required, testId, value }: InputProps) => {
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    onChange(event.target.value);
+  };
+
+  const handleClear = () => {
+    onChange('');
+  };
+
+  return (
+    <FormControl isDisabled={disabled} isInvalid={!!error} isRequired={required} test-id={testId}>
+      <FormLabel
+        background="dolbyNeutral.800"
```

Comment:
Since it's a general input component and we probably could use it in other apps, so I recommend passing colors by arguments or define them in `dolbyio-theme`

Reply:
I updated it so the background of the label and input inherit from its parent. This should allow us to use along any colour scheme, light or dark. I've also included override props `inputProps` and `labelProps` so the user has the option to granularly set styles.